mdata/store_cassandra.go

```diff
@@ -435,7 +446,12 @@ func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]ch
 	results := make(chan outcome, numQueries)
 	for i := range crrs {
 		crrs[i].out = results
-		c.readQueue <- crrs[i]
+		select {
+		case c.readQueue <- crrs[i]:
+		default:
+			cassReadQueueFull.Inc()
+			numQueries--
```

Comment:
this will cause the range loop on the channel at line #459 to never exit.  as no result will be pushed down the channel for this readRequest, seen will never == numQueries and so the channel wont be closed